### PR TITLE
i18n(zh-cn): translate recipe for using `getImage()` to create custom component

### PR DESCRIPTION
**Pull request by 100gle**
#### Description (required)

- translate recipe for using `getImage()` to create custom component
- update corresponsive content in `images.mdx`

Note that the updates in #5210 has synced in this PR, so this PR should be merged after #5210

#### Related issues & labels (optional)

#5042

### CODE REVIEW

File: src/content/docs/zh-cn/recipes/build-custom-img-component.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        width: 800,
+        height: 200,
+    });
+    ---
+
+    ```
+
+4. 创建一个 `<picture>` 元素，它根据你期望的媒体查询生成包含不同图片的 `srcset`。
+
+   ```astro title="src/components/MyCustomImageComponent.astro" ins={28-32}
```
Comment (liruifengv) [inline src/content/docs/zh-cn/recipes/build-custom-img-component.mdx:77]: 看一下语法高亮怎么丢了，好像英文也有问题

Comment (100gle) [reply src/content/docs/zh-cn/recipes/build-custom-img-component.mdx]: 缩进少了空格

File: src/content/docs/zh-cn/recipes/build-custom-img-component.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    ```astro title="src/components/MyCustomImageComponent.astro" 
+    ---
+     import { getImage } from "astro:assets";
+    ---
+
+    ```
+
+2. 创建一个新的组件来处理自定义图片。`MyCustomComponent.astro` 将会从 `Astro.props` 中接收到三个 `props`。`mobileImgUrl` 和 `desktopImgUrl` 属性用于在不同的视口尺寸下创建图片，而 `alt` 属性用于设置图片的替代（alt）文本。这些属性将会在你渲染自定义图片组件的任何地方被传递。添加以下导入语句并定义你在组件中将使用的属性，你也可以使用 TypeScript 为属性添加类型定义。
+
+   ```astro title="src/components/MyCustomImageComponent.astro" ins={3, 11}
```
Comment (liruifengv) [inline src/content/docs/zh-cn/recipes/build-custom-img-component.mdx:27]: 这里也一样

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**